web_switch_company_warning/static/src/js/switch_company_warning_worker.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+//Show a big banner in the top of the page if the company has been
+//changed in another tab or window (in the same browser)
+
+var con = [];
+var lastCtx = null;
+
+addEventListener("connect", function(ee) {
+    var port = ee.ports[0];
+    con.push(port);
+
+    port.onmessage = function (e) {
+        var newCtx = e.data;
+
+        if (lastCtx && newCtx !== lastCtx) {
+            con.forEach(function (eport) {
+                eport.postMessage({type: "newCtx", "newCtx": newCtx, "lastCtx": lastCtx});
```

Comment:
[W7903(javascript-lint), ] Line 17 exceeds the maximum line length of 80. [Error/max-len]